packages/data-models/src/lemma.js

```diff
@@ -45,7 +45,12 @@ class Lemma {
   }
 
   get displayWord () {
-    return this.word.replace(/\d+$/, '')
+    return Lemma.getDisplayWord(this.word)
+  }
+
+  static getDisplayWord (word) {
+    // In the word list we meed to format a word that is in a string format so the getter of the Lemma won't work there
+    return word.replace(/\d+$/, '')
```

Comment:
If to be honest I didn't understand the sense of this refactoring.
The method display word could depend not only on the word of the Lemma, but other properties like language.
By its sense it is not a method of abstract lemma - it is a getter of the current lemma with its current word.
Do you suggest to use this method for any Strings to remove digits?

Reply:
Please see my other comment: https://github.com/alpheios-project/alpheios-core/pull/630#discussion_r566810975

I agree that it could depend on language too, but in that case I think we can add a `language` parameter to the static method and that would solve it.

Do you have any suggestions how to do it differently? I think it's not just about removing the trailing digits (we could use a local method in the wordlist for that), it about applying the same formatting rules to words of lemmas across the app. So I think those rules should be defined in the central location. Once the rules are changed, the change should be applied to all parts of the applications.